knotx-it-tests/src/test/java/io/knotx/example/monolith/SampleApplicationIT.java

```diff
@@ -83,6 +83,12 @@ public void whenRequestingLocalSimplePageWithGet_expectLocalSimpleHtml(TestConte
     testGetRequest(context, LOCAL_REQUEST_URI, "localSimpleResult.html");
   }
 
+  @Test
+  @KnotxConfiguration("knotx-test-app-custom-symbol.json")
```

Comment:
It may be worth to add one additional tests - set custom symbol, but also use the default in the processed snippet markup. Expect custom symbols processed, but `{{` and `}}` left for latter processing by any other scripts/engines (e.g. angular).

Reply:
Fixed